samples/snippets/create_key_asymmetric_decrypt.py

```diff
@@ -44,7 +45,11 @@ def create_key_asymmetric_decrypt(project_id, location_id, key_ring_id, id):
         'purpose': purpose,
         'version_template': {
             'algorithm': algorithm,
-        }
+        },
+
+        # Optional: customize how long key versions should be kept before
+        # destroying.
+        'destroy_scheduled_duration': duration_pb2.Duration().FromSeconds(60*60*24)
```

Comment:
Is this the best way to construct a proto duration? I'm not familiar with the python best practices here.

Reply:
It might be a tad more readable to use `fromTimeDelta` 
```suggestion
        'destroy_scheduled_duration': duration_pb2.Duration().FromTimeDelta(datetime.timedelta(days=1)
```

Reply:
Done